src/foundry/client/pixi/core/interaction/control-icon.d.mts

```diff
@@ -77,13 +67,66 @@ declare global {
   }
 
   namespace ControlIcon {
+    interface Any extends AnyControlIcon {}
+    type AnyConstructor = typeof AnyControlIcon;
+
+    /** @internal */
+    type _Options = InexactPartial<{
+      /**
+       * @defaultValue `40`
+       * @remarks Can't be `null` as it only has a signature-provided default
+       * @privateRemarks It *could* be `null`, as in all the places its used in math, coercing it to `0` does not
+       * produce errors, but it pollutes the `this.size` type unnecessarily and is obviously not intended.
```

Comment:
Given that these aren't stripped out I don't see the point of putting both.

I also am not sure `| null` is enough to count as polluting unless `undefined` is measurably different or something.